api/src/main/kotlin/edu/wgu/osmt/elasticsearch/SearchController.kt

```diff
@@ -333,23 +333,36 @@ class SearchController @Autowired constructor(
                 searchSimilarSkills(apiSimilaritySearch).body?.map { ApiSkillSummaryV2.fromLatest(it) }
             )
     }
-    
+
     @PostMapping(path = [
         "${RoutePaths.API}${RoutePaths.API_V2}${RoutePaths.SEARCH_SIMILARITIES}",
-        "${RoutePaths.API}${RoutePaths.API_V3}${RoutePaths.SEARCH_SIMILARITIES}",
         "${RoutePaths.API}${RoutePaths.UNVERSIONED}${RoutePaths.SEARCH_SIMILARITIES}"
     ],
         produces = [MediaType.APPLICATION_JSON_VALUE])
-    @ResponseBody
-    fun similarSkillWarnings(
-        @RequestBody(required = true) similarities: Array<ApiSimilaritySearch>
-    ): HttpEntity<List<Boolean>> {
+    fun similarSkillWarningsV2(@RequestBody(required = true) similarities: Array<ApiSimilaritySearch>): HttpEntity<List<Boolean>> {
```

Comment:
I dont think we would need to tag this to be version 2. since v2 == 2.5 release. I guess we should just refactor the original one.

Reply:
I understand that v2 is deprecated, so new changes are only added to v3.
V2 is still returning an array of boolean, and v3 is returning similarities.

Reply:
Yeah, but I am not sure if this will be released in v3 and we are pushing more things to be only in v2. so we need to make sure that this is going to be only in v2, what are the plans @Corpratespaz ?

Reply:
we definitely need to keep this boolean return endpoint, but I am not sure if we will just keep it in v2.